### PR TITLE
Add closed form expectation for Gamma

### PR DESCRIPTION
**Pull request by Nimrais**

### CODE REVIEW

File: src/Gamma/Gamma.jl [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    ]
+end
+
+function mean(strategy::ClosedFormExpectation, ::ComposedFunction{Power{Val{3}}, typeof(log)}, q::Gamma)
+    Elogx = mean(strategy, log, q)
+    Elog2x = mean(strategy, Square() ∘ log, q)
+    return polygamma(2, shape(q)) + 3*Elogx * Elog2x - 2*Elogx^3
+end
+
+function mean(strategy::ClosedFormExpectation, p::ComposedFunction{typeof(log), ExpLogSquare{T}}, q::Gamma) where {T}
```
Comment (Raphael-Tresor) [inline src/Gamma/Gamma.jl]: Here you using `p` for the second argument name and in the next function you using `f` better to choose only one name for the argument and having a strong justification for changing it.

File: src/Gamma/Gamma.jl [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    μ, σ = p.inner.μ, p.inner.σ
+    Elogx = mean(strategy, log, q)
+    Elog2x = mean(strategy, Square() ∘ log, q)
+    return -1/(2*σ^2)*(μ^2 - 2*μ*Elogx + Elog2x)
+end
+
+function mean(strategy::ClosedFormExpectation, f::Logpdf{LogNormal{T}}, q::Gamma) where {T}
```
Comment (Raphael-Tresor) [inline src/Gamma/Gamma.jl:51]: Here you naming the second argument `f` , see https://github.com/biaslab/ClosedFormExpectations.jl/pull/7/files#r1570196488

File: src/Gamma/Gamma.jl (hunk 1 above)
Comment (Nimrais) [reply src/Gamma/Gamma.jl]: oh, I see you are right. I will change it to `f` then.

Comment (Nimrais) [reply src/Gamma/Gamma.jl]: I have updated the branch, I have changed the notation to make it more consistent.

**Review by Raphael-Tresor — APPROVED** [review]

**Review by Raphael-Tresor — APPROVED** [review]

**Review by Raphael-Tresor — APPROVED** [review]

**Review by Raphael-Tresor — APPROVED** [review]

**Review by Raphael-Tresor — APPROVED** [review]

**Pull request merged**